CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.4.0
```

Comment:
Also, update the [README usage section](https://github.com/HewlettPackard/oneview-chef#usage) to be `depends 'oneview', '~> 1.4'`